app/vue/src/server/babel_config.test.js

```diff
@@ -1,110 +0,0 @@
-import loadBabelConfig from './babel_config';
```

Comment:
Can you explain please the babel changes? Like why don't we need these tests?

Reply:
The only thing it tested was the addition of `babel-plugin-react-docgen` which is useless in non-react codebase